deepinterpolation/cli/fine_tuning.py

```diff
@@ -126,6 +133,59 @@ def run(self):
             test_generator=data_test_generator
         )
 
+    def _maybe_load_movie_into_memory(
+        self,
+        train_generator_json_path: Path,
+        test_generator_json_path: Path
+    ) -> Optional[Dict[int, np.ndarray]]:
+        """
+        Loads movies into memory if `load_movie_into_memory` is `True`.
+
+        Parameters
+        ----------
+        train_generator_json_path
+        test_generator_json_path
+
+        Returns
+        -------
+        Map from ophys experiment id to movie, if `load_movie_into_memory` is
+        `True`, otherwise `None`
+        """
+        if not self.args["finetuning_params"]["load_movie_into_memory"]:
+            return None
+
+        movs = {}
+        with open(train_generator_json_path) as f:
+            train_generator_config = json.load(f)
+        with open(train_generator_config["train_path"]) as f:
+            train_data = json.load(f)
+
+        with open(test_generator_json_path) as f:
+            test_generator_config = json.load(f)
+        with open(test_generator_config["train_path"]) as f:
+            test_data = json.load(f)
+
+        train_ophys_experiment_ids = list(train_data.keys())
+        test_ophys_experiment_ids = list(test_data.keys())
+
+        if sorted(train_ophys_experiment_ids) != \
+                sorted(test_ophys_experiment_ids):
+            raise NotImplementedError('Different ophys experiments in '
+                                      'train/test not supported when using '
+                                      '`load_movie_into_memory`')
+        ophys_experiment_ids = train_ophys_experiment_ids
+
+        if len(ophys_experiment_ids) > 1:
+            self.logger.warning(
+                'load_movie_into_memory will use a lot of '
+                'memory if there are multiple movies. Are you sure?')
+        for experiment_id in ophys_experiment_ids:
+            self.logger.info(f'Loading {experiment_id} into memory')
+            with h5py.File(
+                    train_data[experiment_id]["path"], "r") as f:
+                movs[experiment_id] = f['data'][()]
```

Comment:
This sets the keys of `movs` as `experiment_id`, whereas the class that it's used references the keys of `movs` as `video_index`.

Are `video_index` the same as `experiment_id`? This seems confusing to have two names referring to the same thing (if they are).